app/main_dev/launch.js

```diff
@@ -796,6 +842,208 @@ export const launchDCRLnd = (
     return resolve(dcrlndCreds);
   });
 
+const Mainnet = 0;
+const Testnet = 1;
+
+export const launchDex = (walletPath, testnet) =>
+  new Promise((resolve, reject) => {
+    if (dex) {
+      return resolve();
+    }
+    try {
+      const dexcRoot = path.join(walletPath, "dexc");
+      const dbPath = path.join(dexcRoot, "db");
+      let error = callDEX("startCore", {
+        dbPath: dbPath,
+        net: !testnet ? Mainnet : Testnet
+      });
```

Comment:
With `logLevel` unset/zero, I believe this is going to start Core in trace level.  I'd suggest debug (1), which is what we use as dexc's default until we hit 1.0.

Reply:
Cool, will change to debug.